### ISSUE TITLE
RIDE always shows status window

### ISSUE THREAD

**Issue by abrudz** [Dyalog/ride#529]
### Describe the issue you are having
`2036⌶0`  turns off "Autostatus", that is, popping up the status window when text is written to it. RIDE seems to ignore that.

[startup.dyalog, the "boot-loader" for Link, relies on this behaviour.](https://github.com/Dyalog/link/blob/f921e28107603836d8dbf8bb8b815591805c5e83/startup.dyalog#L9)

### Did you connect to an already running interpreter or start the interpreter from RIDE?
Start an interpreter

### How do you reproduce the issue?
```
      2036⌶0
1
      2⎕FIX':class' ':endclass'
DOMAIN ERROR: unnamed namespaces are not permitted when fixing multiple objects
      2 ⎕FIX':class' ':endclass'
        ∧
```
![image](https://user-images.githubusercontent.com/10961427/55530189-f792f100-569c-11e9-838b-263fe0e08bb7.png)

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: 4.2.3370
  Platform: Win32
  Date: 2018-07-10 16:00:02 +0200
  Git commit: a9571b73a3c1fe8466c0db329952376414b9e7e1
  Preferences:{
    "floatSingle":"0",
    "floating":"1",
    "ime":"0",
    "kbdLocale":"en_GB",
    "keys":"{\"FX\":[\"Ctrl-S\"]}",
    "lbarOrder":"← +-×÷*⍟⌹○!? |⌈⌊⊥⊤⊣⊢ =≠≤<>≥≡≢ ∨∧⍲⍱ ↑↓⊂⊃⊆⌷⍋⍒ ⍳⍸∊⍷∪∩~ /\\⌿⍀ ,⍪⍴⌽⊖⍉ ¨⍨⍣.∘⍤@ ⍞⎕⍠⌸⌺⌶⍎⍕ ⋄⍝→⍵⍺∇& ¯⍬ ",
    "pfkeys":"[\"\",\"\",\"\",\"\",\"\",\"<ZMI><TC>\",\"\",\"\",\"\",\"\",\"<BP><RM><TC><BP>\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\"]",
    "selectedExe":"C:\\Program Files\\Dyalog\\Dyalog APL-64 18.0 Unicode\\dyalog.exe",
    "squiggleTips":"1",
    "zoom":"6"
  }

Interpreter:
  Version: 18.0.35814
  Platform: Windows-64
  Edition: Unicode/64
  Date: Mar  7 2019 at 21:03:53
```

**Comment by abrudz** [comment]
This may require amending the protocol, as I see [no option](https://github.com/Dyalog/ride/blob/master/docs/protocol.md#status-window) to add output without showing the window.

**Comment by e9gille** [comment]
Yes, either protocol change or the interpreter simply could omit posting the message if user turns of the feature.

**Comment by abrudz** [comment]
In the IDE, the message is still written if auto-status is off, but the window isn't shown, so the user can manually go and inspect what happened. Anyway, I suggest adding `show: false`

**Comment by abrudz** [comment]
I've raised [issue 17116](http://mantis.dyalog.com/view.php?id=17116) for the interpreter side of things.

**Comment by jayfoad** [comment]
I suggest that the interpreter has no business telling RIDE whether to show the status window. That should be a RIDE preference.

**Comment by e9gille** [comment]
> That should be a RIDE preference.

Sure, but at the moment the developer can suppress the window using the I-beam. I don't know where to draw the line, but at the moment RIDE isn't notified of this. We could add a preference in RIDE, set a sensible default (off?) and ignore the I-beam?

**Comment by abrudz** [comment]
@jayfoad No, the interpreter is not telling RIDE to show the window or not, it is just telling RIDE what the current interpreter status (the one set with an I-beam) is. RIDE (or whatever is at the other end of the line) can of course do whatever it wants with that information.
